data/templates/esx-ks

```diff
@@ -245,7 +245,8 @@ wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%
 esxcli system shutdown reboot -d 10 -r "Rebooting after first boot host configuration"
```

Comment:
Not sure why this is showing as a diff. I got the change from master.
https://github.com/RackHD/on-http/blob/master/data/templates/esx-ks#L248